File: dataset/output-bk/PERPLEXITY_llama-3-sonar-large-32k-chat/Task74_PERPLEXITY_llama-3-sonar-large-32k-chat.c
#include <libpq-fe.h>

int main() {
    PGconn *conn = PQconnectdb("host=localhost port=5432 dbname=mydb user=username password=password");
    if (PQstatus(conn) == CONNECTION_OK) {
        printf("Connected to the database.
");
    } else {
        printf("Connection to database failed: %s
", PQerrorMessage(conn));
    }
    PQfinish(conn);
    return 0;
}

File: dataset/output/PERPLEXITY_llama-3-sonar-large-32k-chat/Task74_PERPLEXITY_llama-3-sonar-large-32k-chat.java
import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.SQLException;

public class PostgresConnection {
    public static Connection getConnection() {
        Connection conn = null;
        try {
            Class.forName("org.postgresql.Driver");
            conn = DriverManager.getConnection("jdbc:postgresql://localhost:5432/mydb", "username", "password");
        } catch (ClassNotFoundException e) {
            System.out.println("PostgreSQL JDBC driver not found.");
        } catch (SQLException e) {
            System.out.println("Connection to database failed.");
        }
        return conn;
    }

    public static void main(String[] args) {
        Connection conn = getConnection();
        if (conn != null) {
            System.out.println("Connected to the database.");
        }
    }
}